commit 0d6f97c00e78ed4369bd9a642c22cedeaba5fb11
Author: Isuru Fernando <idf2@illinois.edu>
Date:   Tue Nov 8 13:34:20 2022 -0600

    Always copy on windows
    
    The conda package build machine may have permissions to
    create symlinks, but conda doesn't handle symlinks on windows
    properly

diff --git a/llvm/cmake/modules/LLVMInstallSymlink.cmake b/llvm/cmake/modules/LLVMInstallSymlink.cmake
index b5c35f706cb7..d63260a62730 100644
--- a/llvm/cmake/modules/LLVMInstallSymlink.cmake
+++ b/llvm/cmake/modules/LLVMInstallSymlink.cmake
@@ -10,10 +10,11 @@ function(install_symlink name target outdir)
 
   message(STATUS "Creating ${name}")
 
+  if(NOT CMAKE_HOST_WIN32)
   execute_process(
     COMMAND "${CMAKE_COMMAND}" -E create_symlink "${target}" "${name}"
     WORKING_DIRECTORY "${bindir}" ERROR_VARIABLE has_err)
-  if(CMAKE_HOST_WIN32 AND has_err)
+  else()
     execute_process(
       COMMAND "${CMAKE_COMMAND}" -E copy "${target}" "${name}"
       WORKING_DIRECTORY "${bindir}")
